### DISCUSSION TITLE
miktex-dvipdfmx need to be repaired

### DISCUSSION THREAD

**Discussion by xacbeyond** [MiKTeX/miktex#982] [Q&A] [answered]
Dear all,
When I was trying to update MikTex, the console showed that miktex-dvipdfmx need to be repaired. I did it and tried to update again, it still showed me it needs to be repaired. Does anyone know how to fix this issue?

Thank you!

**Comment by edocevoli** [comment, marked as answer]
Thank you, this has been fixed. Please check for updates tomorrow.

**Reply by xacbeyond** [reply]
> Thank you, this has been fixed. Please check for updates tomorrow.

Issue fixed! Thank you very much!
